coala_quickstart/interaction/Logo.py

```diff
@@ -0,0 +1,60 @@
+from coala_quickstart.Strings import coala_bear_logo, welcome_messages
+
+
+def text_wrap(*args, delimiter=' ', end='\n', limit=80):
+    """
+    Breaks it wordwise when going further than ``limit`` chars.
```

Comment:
this can be nicely doctested, can you do that?

```
>>> text_wrap("I", "love", "gooooooooooood", "wraps", end='!', limit=6)
I love
gooooooooooood
wraps!
```